app/Nova/Event.php

```diff
@@ -88,14 +89,19 @@ public function fields(Request $request): array
                 return route('events.rsvp', ['event' => $this->id]);
             })->onlyOnDetail(),
 
+            MorphMany::make('Remote Attendance Links', 'remoteAttendanceLinks')
+                ->canSee(static function (Request $request): bool {
+                    return $request->user()->can('read-remote-attendance-links');
+                }),
+
             self::metadataPanel(),
 
             HasMany::make('RSVPs')
                 ->canSee(static function (Request $request): bool {
                     return $request->user()->can('read-rsvps');
                 }),
 
-            HasMany::make('Attendance')
+            MorphMany::make('Attendance')
```

Comment:
Was this broken before? 🤔

Reply:
No; the classes are equivalent. This is more semantically correct.